Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -30,17 +30,17 @@
 from .branch import Branch
 from .pull_request import PullRequest
 
-from webkitbugspy import Tracker
-from webkitcorepy import arguments, run, Terminal
+from webkitbugspy import Tracker, radar, bugzilla
+from webkitcorepy import arguments, run, Terminal, string_utils
 from webkitscmpy import local, log, remote
 from ..commit import Commit
 
 
 class Revert(Command):
```

Comment:
I think the changes to this class should be separated from the `prepare-commit-msg` changes. It may make sense to develop them as a stacked PR for now, but I think they should ultimately land separately. My reasoning for this is that I think that we should make an effort to make `git revert` do something mostly correct. While we can certainly encourage usage of `git-webkit revert`, I know there are some workflows where contributors will use `git revert`, and I'd like to be able to standardize those commits as well.